src/main/resources/openapi.json

```diff
@@ -1,286 +1,1279 @@
+                    }
+                }
+            ]
         },
-        {
-          "name": "name",
-          "in": "path",
-          "required": true,
-          "schema": {
-            "type": "string"
-          }
-        }
-      ]
-    },
-    "/consumers/{groupid}/instances/{name}/subscription": {
-      "post": {
-        "operationId": "subscribe",
-        "responses": {
-          "200": {
-            "description": "OK"
-          }
-        }
-      },
-      "delete": {
-        "operationId": "unsubscribe",
-        "responses": {
-          "200": {
-            "description": "OK"
-          }
-        }
-      },
-      "parameters": [
-        {
-          "name": "groupid",
-          "in": "path",
-          "required": true,
-          "schema": {
-            "type": "string"
-          }
+        "/consumers/{groupid}/instances/{name}": {
+            "delete": {
+                "tags": [
+                    "Consumers"
+                ],
+                "description": "Deletes a specified consumer instance. The operation MUST be performed on the the Kafka Bridge instance that contains the consumer.",
+                "operationId": "deleteConsumer",
+                "responses": {
+                    "204": {
+                        "description": "Consumer removed successfully."
+                    },
+                    "404": {
+                        "description": "The specified consumer instance was not found.",
+                        "content": {
+                            "application/json": {
+                                "schema": {
+                                    "$ref": "#/components/schemas/Error"
+                                },
+                                "examples": {
+                                    "response": {
+                                        "value": {
+                                            "error_code": 404,
+                                            "message": "The specified consumer instance was not found."
+                                        }
+                                    }
+                                }
+                            }
+                        }
+                    }
+                }
+            },
+            "parameters": [
+                {
+                    "name": "groupid",
+                    "in": "path",
+                    "description": "ID of the consumer group to which the consumer belongs.",
+                    "required": true,
+                    "schema": {
+                        "type": "string"
+                    }
+                },
+                {
+                    "name": "name",
+                    "in": "path",
+                    "description": "Name of the consumer that you want to delete.",
+                    "required": true,
+                    "schema": {
+                        "type": "string"
+                    }
```

Comment:
So if the delete request requires both the name and the group id I assume that means that consumer names are unique within the scope of a consumer id, rather than within the scope of the whole bridge. You should probably say that in the docs for the create endpoint.

Reply:
The name is unique within the scope of the consumer group.